recipes/tensap/setup.patch

```diff
@@ -0,0 +1,825 @@
+From a5d26f4de04f2868f03d5a7932ca64d9d127c93a Mon Sep 17 00:00:00 2001
```

Comment:
This file isn't being used - is that because there is a new release?

Reply:
oh, my mistake